src/proxy/routes-engine/rules/mimeTypes.ts

```diff
@@ -0,0 +1,13 @@
+import http from "http";
+
+// See: https://docs.microsoft.com/en-us/azure/static-web-apps/configuration
+export const mimeTypes = async (req: http.IncomingMessage, res: http.ServerResponse, mimeTypes: SWAConfigFileMimeTypes) => {
+  if (req.url?.includes(".")) {
+    const fileExtentionFromURL = req.url?.split(".").pop();
+    const overrideMimeType = mimeTypes[`.${fileExtentionFromURL}`];
```

Comment:
```suggestion
    const overrideMimeType = mimeTypes?.[`.${fileExtentionFromURL}`];
```

I hit a null ref here.